distribution/docs/src/main/resources/_contents/_installing/installing-intro-contents.adoc

```diff
@@ -0,0 +1,367 @@
+
+|`lib`
+|The system's bootstrap libraries. Includes the `${branding-lowercase}-branding.jar` file which is used to brand the system console with the ${branding} logo.
+
+|`licenses`
+|Licensing information related to the system.
+
+|`system`
+|Local bundle repository. Contains all of the JARs required by ${branding}, including third-party JARs.
+
+|===
+
+==== Deployment Guidelines
+
+${branding} relies on the Directory Permissions of the host platform to protect the integrity of the ${branding} during operation.
+System administrators MUST perform the following steps prior to deploying bundles added to the ${branding}.
+
+. Check the available storage space on the system to ensure the deployment will not exceed the available space.
+. Set maximum storage space on the `${branding}_HOME/deploy` and `${branding}_HOME/system` directories to restrict the amount of space used by deployments.
+. Do not assume the deployment is from a trusted source; verify its origination.
+
+==== Directory Permissions
+
+Restrict access to sensitive files by ensuring that the only users with access privileges are administrators.
+
+===== Directory Permissions on Windows
+
+. Right-click on the file or directory noted below then select *Full Control -> Administrators -> System*.
+. Click *Properties -> Security -> Advanced* and select `Creator Owner` for `${branding}_HOME` (e.g., `C:\ddf`).
+. Restrict access to sensitive files by ensuring that only *System* and *Administrators* have Full Control to the below files by right-clicking on the file or directory below then selecting *Properties -> Security -> Advanced*
+. Delete any other groups or users listed with access to `${branding}_HOME/etc` and `${branding}_HOME/deploy`.
+
+===== Directory Permissions on *NIX
+
+Protect the ${branding} from unauthorized access.
+////
+Old version
+. As root, change the owner and group of critical ${branding} directories to the `NON_ROOT_USER`.
++
+[WARNING]
+====
+A `NON_ROOT_USER` (e.g., ${branding-lowercase}) is recommended for installation.
+====
++
+[source, bash, linenums]
+----
+chown -R NON_ROOT_USER ${branding}_HOME ${branding}_HOME/etc ${branding}_HOME/data
+chgrp -R NON_ROOT_USER ${branding}_HOME/etc ${branding}_HOME/data
+chmod -R og-w ${branding}_HOME/etc ${branding}_HOME/data
+----
++
+. Restrict access to sensitive files by ensuring that the only users with “group” permissions (e.g., ${branding-lowercase}-group) have access.
+. Execute the following command on the above files (examples assume `${branding}_HOME` is `<${branding}_HOME>`):
++
+[source,bash]
+----
+chmod -R o <${branding}_HOME>
+----
++
+. As the application owner (e.g., ${branding-lowercase} user), restrict access to sensitive files.
++
+[source,bash,linenums]
+----
+chmod 640 <${branding}_HOME>/etc
+chmod 640 <${branding}_HOME>/deploy
+----
+////
+// New Version
+* Change ownership of ${branding}_HOME
```

Comment:
Shouldn't we put all the `DDF_HOME` within `<>` to be consistent with other directory references such as `<INSTALL_DIRECTORY>`?